### PR TITLE
Review Screen

### PR DESCRIPTION
**Pull request by carizzuti**
Created Review Screen, currently using dummy data.

Screenshot_1651518424

### CODE REVIEW

File: mobile/src/components/textInput/TextInput.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const TextInput: FC<TTextInputProps> = ({
   const [text, setText] = useState('');
   const [secureTextEntry, setSecureTextEntry] = useState(Boolean);
   const textInputRef = useRef(null);
-  let max = type === 'phone' ? 17 : 30;
+  let max = type === 'phone' ? 17 : 300;
```
Comment (raynesmike) [inline mobile/src/components/textInput/TextInput.tsx:26]: Why did you change this charlie? this was meant for the logic for the length of phone number.

Comment (raynesmike) [inline mobile/src/components/textInput/TextInput.tsx:26]: I should probably update this textInput. and handle everything inside the switch.

**Comment by Dulaya** [comment]
- Your ChevronLeft icon is a little off to the right.
- Okay to remove `TReviewScreenProps` in Review.type.ts if you're not using it: 

         export type TReviewScreenProps = StackScreenProps<
            TAppNavigationRoutes,
            EAppNavigationRoutes.REVIEW
         >;
- Optional: 
   - Shorten the review texts, or use ... That way, you can see more reviews. 
   - Your Review Card has unnecessary margin. It should be aligned with the TextInput:
image
- No major issues besides styling. Good job. ✔️

**Review by Dulaya — APPROVED** [review]

File: mobile/App.tsx [hunk 2]
```diff
@@ -1,25 +1,14 @@
 import React, {useEffect} from 'react';
 import SplashScreen from 'react-native-splash-screen';
-import {ReviewCard} from './src/components/reviewCard/ReviewCard';
 import {Router} from './src/navigation';
-import {TReview} from './src/types/data';
 
 const App = () => {
   useEffect(() => {
     setTimeout(() => {}, 1000);
```
Comment (lasharela) [inline mobile/App.tsx:7]: try to remove

**Review by lasharela — CHANGES_REQUESTED** [review]

**Review by Josetf2396 — APPROVED** [review]

**Review by Nap1400 — APPROVED** [review]

**Pull request merged**